src/vos/vos_tree.c

```diff
@@ -779,6 +796,7 @@ static btr_ops_t singv_btr_ops = {
 	.to_rec_free		= svt_rec_free,
 	.to_rec_fetch		= svt_rec_fetch,
 	.to_rec_update		= svt_rec_update,
+	.to_rec_corrupt		= svt_rec_corrupt,
```

Comment:
can we not do this through rec_update for single value?   It's only needed for single value, right?

Reply:
agreed, looks it's not a common thing, could be done via svt_rec_update()?

Reply:
Well, I tried that, but it "/** Disallow[s] same epoch overwrite */", and then appears to frees & allocate mem.